static/app/components/idBadge/baseBadge.tsx

```diff
@@ -79,7 +79,7 @@ const DisplayNameAndDescription = styled('div')`
 const DisplayName = styled('span')`
   overflow: hidden;
   text-overflow: ellipsis;
-  line-height: 1.2;
```

Comment:
This shouldn’t have an effect on the other instances, will double check the snapshots. But this is why the text was misaligned when the font-size property is changed.